templates/todo/tag_confirm_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h1>Delete tag</h1>
+
+
+  <p>Are you sure you want to delete the tag {{ tag }}</p>
+
+  <form action="" method="post">
+  {% csrf_token %}
+      <input type="submit" value="Yes, delete" class="btn btn-danger">
```

Comment:
It should also have button with option to cancel deletion.

`<a href="{% url 'todo:tag-list' %}" class="btn btn-secondary">Cancel</a>`